sdk/maps/azure-maps-route/setup.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#--------------------------------------------------------------------------
+
+import re
+import os.path
+from io import open
+from setuptools import find_packages, setup
+
+# Change the PACKAGE_NAME only to change folder and different name
+PACKAGE_NAME = "azure-maps-route"
+PACKAGE_PPRINT_NAME = "Azure Maps Route"
```

Comment:
Duplicate "Azure" in line 16 and line 56, should remove "Azure" from line 16 according to azure-template

Reply:
Same for search and render package